lib/widgets/profile.dart

```diff
@@ -33,6 +35,37 @@ class ProfilePage extends StatelessWidget {
       page: ProfilePage(userId: userId));
   }
 
+  /// The given user's real email address, if known, for displaying in the UI.
+  ///
+  /// Returns null if self-user isn't able to see [user]'s real email address.
+  ///
+  /// **Note:** Starting from Zulip version 7.0 (FL 163), there's a change in
+  /// the API about how self-user can access [user]'s real email address.
+  /// Search for "delivery_email" in https://zulip.com/api/register-queue.
+  // TODO(server-7): remove; use user-level setting
+  String? _getDisplayEmailFor(User user, {required PerAccountStore store}) {
+    if (store.account.zulipFeatureLevel >= 163) {
```

Comment:
```suggestion
  String? _getDisplayEmailFor(User user, {required PerAccountStore store}) {
    if (store.account.zulipFeatureLevel >= 163) { // TODO(server-7)
```

Having the comment right at the condition is helpful for seeing that the condition is covered by a TODO-server comment.

The instruction "use user-level setting" isn't right for this spot — we won't be looking at the user's individual `emailAddressVisibility` setting, but rather just looking at their `deliveryEmail`. After all, that's what this code already does for modern servers. If there were something else we should be doing in the future, then for modern servers we should be doing that in the first place.